### DISCUSSION TITLE
NRQCD in FeynRules

### DISCUSSION THREAD

**Discussion by Abdulrahma-M** [FeynCalc/feyncalc#141] [Q&A] [answered]
I am trying to create an NRQCD model to study Exotic Higgs decays in MG5, Using FeynRules, FeynCalc 9.3 for the D-1 Cartesian functions and FeynOnium to break the Lorentz Covariance and ease the Cartesian tensor manipulation. Yet it doesn't seem intuitive to me how i can best use FeynOnium in a Lagrangian situation, in regards to replacing a Feynrule Dirac Spinor with a Pauli Spinor and handling the Chrome-Electric/Magnetic fields . Any insight would be much appreciated.

**Comment by vsht** [comment, marked as answer]
As you might have inferred from the examples shipped with FeynOnium, the high energy part
of the matching can be pretty well automatized, since for that you need only the Lorentz covariant
amplitudes resulting from your Feynman diagrams (generated e.g. with FeynArts). Those can
be then manipulated using FeynOnium routines.

As for the low-energy part (in this case NRQCD), in the current state one has to write down
the relevant nonrelativistic amplitudes by hand. 

FeynArts, FeynRules and MadGraph simply cannot deal with genuine nonrelativistic Lagrangians
and the corresponding Feynman rules. So here you would need to go with QGRAF and a set
of self-derived Feynman rules. As of now I'm not really aware of any automatic setup for that part.

**Reply by Abdulrahma-M** [reply]
Thank you for you help
